Demo/Demo/Models/PaymentTokenRequest.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CorePayments
+
+struct PaymentTokenRequest {
+    
+    let setupToken: String
+    
+    var path: String {
+        "/payment_tokens/"
+    }
+    
+    var method: CorePayments.HTTPMethod {
+        .post
+    }
+    
+    var headers: [CorePayments.HTTPHeader: String] {
```

Comment:
I know we use `CorePayments.HTTPMethod` and `CorePayments.HTTPHeader` in a few places in the demo app, but are these exposed for merchant consumption or should they be `nodoc`ed and not used in the demo app?

Reply:
Great catch! Those shouldn't be used by merchants

Reply:
Do you want me to fix it for` ClientIDRequest` as well?